packages/cli-kube/src/handlers/register.ts

```diff
@@ -3,10 +3,56 @@
 //
 
 import assert from 'assert';
+import got from 'got';
 
-import { DXN, IRegistryClient, RecordKind } from '@dxos/registry-client';
+import { CID, DomainKey, DXN, IRegistryClient, RecordKind } from '@dxos/registry-client';
 
 export const KUBE_DXN_NAME = 'dxos:type.kube';
+export const SERVICE_TYPE_DXN = 'dxos:type.service';
+
+interface RegisterServiceOptions {
+  registryClient: IRegistryClient,
+  kubeCID: CID,
+  domainKey: DomainKey,
+  url: string
+}
+
+const getServiceTypeCID = async (registryClient: IRegistryClient, serviceName: string) => {
+  // Checking for specific type, like dxos:type.service.app-server
+  const cid = await registryClient.resolveRecordCid(DXN.parse(SERVICE_TYPE_DXN + '.' + serviceName));
+  if (cid) {
+    const record = await registryClient.getTypeRecord(cid);
+    if (record) {
+      return cid;
+    }
+  }
+  // Can't resolve specific type, return default
+  const defaultCID = await registryClient.resolveRecordCid(DXN.parse(SERVICE_TYPE_DXN));
+  assert(defaultCID);
+  console.log('Default type');
+  return defaultCID;
+};
+
+const registerServices = async (options: RegisterServiceOptions) => {
+  const reponse = await got(`${options.url}/kube/services`);
```

Comment:
please move out `/kube/services` to a separate const ;
if http request unsuccessful, throw nice err saying KUBE is not reachable by that url